react/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment, FC, useRef, useEffect } from 'react'
+import { generateBlockClass, BlockClass } from '@vtex/css-handles'
+//@ts-ignore
```

Comment:
Add definition for css files 
https://github.com/vtex-apps/base-store-component/blob/master/react/typings/css.d.ts